target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,129 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hootoo,tm05", "ralink,mt7620n-soc";
+	model = "HooToo TM05";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		green {
+			label = "tm05:green";
+			gpios = <&gpio2 0 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+			// Enabled below line, if flashing LED desired
+			//linux,default-trigger = "timer";
+		};
+
+		blue {
+			label = "tm05:blue";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&gpio0 {
+	status = "okay";
+};
```

Comment:
You can remove this block, gpio0 is already enabled.

Reply:
Sure, will do.

Reply:
Actually, we do not need gpio0 (as you show below for gpio0), so you could put status = "disabled" here and check ...

Reply:
Yep, removed it - NP at all. Thanks!